ci/Jenkinsfile_utils.groovy

```diff
@@ -18,18 +18,25 @@
 // under the License.
 
 // initialize source codes
-def init_git() {
+// Clean - 0: None, 1: xdf, 2: xdff
+def init_git(clean=2) {
   deleteDir()
   retry(5) {
     try {
       // Make sure wait long enough for api.github.com request quota. Important: Don't increase the amount of
       // retries as this will increase the amount of requests and worsen the throttling
       timeout(time: 15, unit: 'MINUTES') {
         checkout scm
-        sh 'git clean -xdff'
+        if(clean == 0)
+          cleanCmd = 'git status'
+        else if(clean == 1)
+          cleanCmd = 'git clean -xdf'
+        else if(clean == 2)
+          cleanCmd = 'git clean -xdff'
+        sh cleanCmd
```

Comment:
Could you elaborate why this is needed? This "super brutal clean" as I like to call it, was requested by @lebeg and he said it's important. Maybe you two could have a discussion there :)

Reply:
Actually, I can remove this because we are no longer going to use the maven-release-plugin following updates as part of #13626